website/docs/r/app_service_certificate_order.html.markdown

```diff
@@ -25,7 +25,7 @@ resource "azurerm_app_service_certificate_order" "example" {
   resource_group_name = "${azurerm_resource_group.example.name}"
   location            = "global"
```

Comment:
this global location confused me...is this some tf doc practice, or does it have special meaning, or is it just a random val to be different than the resource group?

Reply:
ah i just tried some other region and sure enough it must be global 
```
Error: Error creating/updating App Service Certificate Order "usos1cer01-inmp-dev" (Resource Group "usos1rgp01-certs"): web.AppServiceCertificateOrdersClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="LocationNotAvailableForResourceType" Message="The provided location 'westus2' is not available for resource type 'Microsoft.CertificateRegistration/certificateOrders'. List of available regions for the resource type is 'global'
```